actions/socket-export-sbom/action.yml

```
@@ -0,0 +1,45 @@
+name: Export SPDX SBOM from Socket.dev API
+description: Export SPDX SBOM from Socket.dev API for a given repository.
+
+inputs:
+  socket_api_token:
+    description: "Socket API token for authentication"
+    required: true
+  socket_base_url:
+    description: "Socket base url"
+    required: true
+    default: "api.socket.dev/v0"
+  socket_org:
+    description: "Socket org name"
+    required: true
+    default: "grafana"
+  output_file:
+    description: "Output file path for the SBOM"
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Go
+      uses: actions/setup-go@v5
```

Pin this action to a hash.
```suggestion
      uses: actions/setup-go@4dc6199c7b1a012772edbd06daecab0f50c9053c # v6.1.0
```